Directory.Build.props

```diff
@@ -4,4 +4,39 @@
     <CheckEolTargetFramework>false</CheckEolTargetFramework>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
+
+  <!-- To reduce build times, we only enable analyzers for the newest TFM -->
```

Comment:
Moved inclusion and configuration of analysis here to share them between all projects